src/database/models/users.js

```diff
@@ -0,0 +1,18 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const users = sequelize.define('users', {
+    firstname: DataTypes.STRING,
```

Comment:
### Critical
- Define this with other constraints, like primary key, unique, foreign key and data types, `NOT NULL` etc.

Reply:
for the primary key it is already defined in the migration file for users. and about the foreign keys we don't have relationships yet we will be using them later in the project